migration/transformation_schema_2_to_1.yaml

```diff
@@ -0,0 +1,134 @@
+# This file provides specific mappings for metric data between Java 2.x to Java 1.x
+# The format should be the same as the otel file_format for schema files.
+file_format: 1.1.0
+# TODO: Fix URL
+schema_url: https://github.com/signalfx/splunk-otel-java/migration/transformation_schema_2_to_1.yaml
+versions:
+  # Java version 2.0.0 is the new version. The following mappings translate
+  # FROM the new/correct/stable names back to the previous names.
+  # The names to use when applying attribute renaming are the NEW names.
+  # <new_name>: <old_name>
+  # We want to translate TO the old names as a migration step
+  2.0.0:
+    metrics:
+      changes:
+        #--------------------------------------------------------------
+        # Memory metrics
+        - rename_metrics:
+            jvm.memory.used:               runtime.jvm.memory.used           # micrometer
+            jvm.memory.committed:          runtime.jvm.memory.committed      # micrometer
+            jvm.memory.limit:              runtime.jvm.memory.max            # micrometer
+            jvm.memory.used_after_last_gc: runtime.jvm.memory.usage.after.gc # micrometer
+#            jvm.memory.init:               process.runtime.jvm.memory.init  # experimental
+        - rename_attributes:
+            attribute_map:
+              jvm.memory.type: area
+              jvm.memory.pool.name: pool
+            apply_to_metrics:
+              - jvm.memory.used
+              - jvm.memory.committed
+              - jvm.memory.limit
+              - jvm.memory.used_after_last_gc
+#              - jvm.memory.init
+
+        #--------------------------------------------------------------
+        # Threads metrics
+        - rename_metrics:
+            jvm.thread.count: runtime.jvm.threads.live # micrometer
```

Comment:
According to our source, `runtime.jvm.threads.daemon` also renames to `jvm.thread.count`.

Reply:
@laurit @breedx-splk Can you confirm? Could you reflect that in the mapping if that's the case?

Reply:
I think it's a problem if we try and map multiple metrics to the same metric.

Reply:
What should we do with `threads.daemon` then?

Reply:
* [Here](https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmThreadMetrics.java#L64) is the definition for `jvm.threads.daemon`
* [Here](https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmThreadMetrics.java#L70) is the definition for `jvm.threads.live`. 
* [Here](https://opentelemetry.io/docs/specs/semconv/runtime/jvm-metrics/#metric-jvmthreadcount) is the definition for otel `jvm.thread.count`. 

So yeah, it's not exactly the same thing, and I'm not convinced there's a clean mapping.